File: cosmic-journal/cosmic-journal-backend/app/utils/moodUtils.py
VALID_MOODS = {"reflective", "joyful", "calm", "excited"}

def validate_mood(mood: str) -> str:
    """Validate a mood tag."""
    if mood and mood not in VALID_MOODS:
        raise ValueError(f"Invalid mood: {mood}. Must be one of {VALID_MOODS}")
    return mood

File: cosmic-journal-frontend/src/utils/moodUtils.js
export const moodOptions = [
  { value: 'reflective', label: 'Reflective' },
  { value: 'joyful', label: 'Joyful' },
  { value: 'calm', label: 'Calm' },
  { value: 'excited', label: 'Excited' },
];

export function getMoodLabel(value) {
  const mood = moodOptions.find((m) => m.value === value);
  return mood ? mood.label : 'Unknown';
}